.circleci/config.yml

```diff
@@ -58,12 +66,34 @@ jobs:
       - setup
       - run: pip install .
       - run: tern -l report -i photon:3.0
+  # Deploy to PyPi
+  pypi_deploy:
+   executor: ubuntu1604
+   steps:
+      - setup
+      - run: pip install .
+      - run: pip install twine
+      # make sure git tag matches release version
+      - run: python setup.py verify
+      - create_pypirc 
+      - run: python setup.py sdist
+      - run: twine upload dist/* 
+
 workflows:
-  # run a full functional test for photonOS
   version: 2
   PRs:
     jobs:
       - linting
       - commit_check
       - security
       - test_changes
+  Release:
+    jobs:
+      - pypi_deploy:
+          filters:
+            # ignore all commits on any branch by default
+            branches:
+              ignore: /.*/
+            # only run on version tags
+            tags:
+              only: /^v[0-9]+(\.[0-9]+)*$/
```

Comment:
Did we decide to stick in a post-deploy job for testing if the pip package worked and have it be dependent on the pypi_deploy job?

Reply:
Yes we did. I thought this functionality would be added after the 0.5.0 release, however, since currently there is nothing to verify on PyPI. What about opening a separate issue for this test script and merging it as a separate PR?

Reply:
That works for me.